react/typings/events.d.ts

```diff
@@ -0,0 +1,8 @@
+export interface EventWithValue extends Event {
```

Comment:
Aren't there other `Events` you could have used? Maybe `React.SyntheticEvent`, `React.MouseEvent`, `React.PointerEvent`, etc...

Reply:
Yes, now I'm using React.ChangeEvent for the event in component RadioGroup from styleguide. But for the component NumericStepper I have to maintain EventWithValue.